desktop/core/src/desktop/js/reactComponents/InputModal/InputModal.tsx

```diff
@@ -67,15 +70,17 @@ const InputModal = ({
       open={showModal}
       title={title}
       secondaryButtonProps={{ disabled: buttonDisabled }}
-      okButtonProps={{ disabled: buttonDisabled }}
+      okButtonProps={{ disabled: buttonDisabled || initialValue === value }}
```

Comment:
Nit: this is a nice little check that you could include your unit test since you are testing for disabled button